system-tests/__snapshots__/screenshots_spec.js

```diff
@@ -131,13 +131,13 @@ Because this error occurred during a \`after each\` hook we are skipping the rem
   -  /XXX/XXX/XXX/cypress/screenshots/screenshots.cy.js/element.png                        (400x300)
   -  /XXX/XXX/XXX/cypress/screenshots/screenshots.cy.js/taking screenshots -- retries     (200x1300)
       each screenshot for up to  XX:XX.png                                                          
-  -  /XXX/XXX/XXX/cypress/screenshots/screenshots.cy.js/retrying-test.png                (1000x1316)
+  -  /XXX/XXX/XXX/cypress/screenshots/screenshots.cy.js/retrying-test.png                 (1000x660)
```

Comment:
Does this pass? I changed sizes on a system test once and had to revert the sizes back because the test wouldn't pass in CI with the new sizes.

Reply:
Looks like it did pass, there is probably something i should have done with the other tests to ignore that part of the diff.